src/libs/SearchQueryUtils.ts

```diff
@@ -335,9 +335,12 @@ function buildQueryStringFromFilterFormValues(filterValues: Partial<SearchAdvanc
         filtersString.push(`${CONST.SEARCH.SYNTAX_ROOT_KEYS.TYPE}:${sanitizedType}`);
     }
 
-    if (status) {
-        const sanitizedStatus = sanitizeSearchValue(status);
-        filtersString.push(`${CONST.SEARCH.SYNTAX_ROOT_KEYS.STATUS}:${sanitizedStatus}`);
+    if (status && status?.length !== 0) {
+        if (typeof status === 'string') {
```

Comment:
I wonder if it would make our life easier, to treat `status` always as an array. I see in more than one place you need to check if its string or array, but then perform essentially the same operation 🤔 

WDYT?

Reply:
We cannot treat status as an array, because of parser logic, it will be easier to add one `if` here than, change parser logic.